GPy/kern/src/multioutput_kern.py

```diff
@@ -59,24 +92,33 @@ def Kdiag(self,X):
         target = np.zeros(X.shape[0])
         [[np.copyto(target[s], kern.Kdiag(X[s])) for s in slices_i] for kern, slices_i in zip(kerns, slices)]
         return target
-
+    
+    def update_gradients_full_wrapper(self, cov_struct, dL_dK, X, X2):
+        gradient = cov_struct['kern'].gradient.copy()
+        cov_struct['update_gradients_full'](dL_dK, X, X2)
+        cov_struct['kern'].gradient += gradient
+    
+    def update_gradients_diag_wrapper(self, kern, dL_dKdiag, X):
```

Comment:
As far as I can see, this is an internal class method. Following the Python name convention, I would name it as '_update_gradients_diag_wrapper'. Similarly, I would name update_gradients_full_wrapper as '_update_gradients_full_wrapper'.

Reply:
Ok, thank you for noticing this. It is now changed!